### ISSUE TITLE
Be consistent with the use of "view modes" or "displays" in the admin UI and also have a consistent UI for them.

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#2342]
@jenlampton in https://github.com/backdrop/backdrop-issues/issues/315#issuecomment-261446864:

> Also, were these supposed to be called "Display modes" in the UI? To decrease confusion with Views?

There are currently two places in the admin UI that I can think of where this applies to:

- The Views UI, and specifically the `views-display-top` "ribbon":

  ![backdrop-views_ui-displays](https://cloud.githubusercontent.com/assets/2423362/20444846/dacff610-ae26-11e6-97d1-7883f3c2df73.png)

- The upcoming UI change that is coming with #315 in the "Manage display" tab of content types:

  ![backdrop-issue315-view_modes_or_displays](https://cloud.githubusercontent.com/assets/2423362/20445055/b746ca92-ae27-11e6-918a-69c151358607.png)

So, I was thinking that it would be more consistent to call all things that display a specific thing in a different fashion "displays". That way, we would have all these things have the label "Displays" in the admin UI, while in documentation we could refer to them as "content type displays", "views displays" and if in the future we add more things that can be displayed in various ways "what-have-you displays".

Also, we all seem to agree that the Views UI is not very user friendly and also not consistent with the rest of the admin UI in the Seven theme. I believe that we won't escape the fact that at some point we need to redesign it. I think that this could be a nice opportunity to gradually start by making this small change:

![backdrop-content_type-displays_as_secondary_tabs](https://cloud.githubusercontent.com/assets/2423362/20448568/c5c70ad0-ae38-11e6-8b50-59364672e6bf.png)

![backdrop-views_ui-displays_as_secondary_tabs](https://cloud.githubusercontent.com/assets/2423362/20448576/ceed641a-ae38-11e6-95ed-ad757ccf676b.png)

**Comment by jenlampton** [comment]
I'm okay with this :)

**Comment by jenlampton** [comment]
"View modes" and "Displays" are very different things, I think we need to be careful not to call them by the same name, lest people become confused.

* A views "Display" is a thing, for example: a page, a block, or an RSS feed. Adding a new "Display" to your view is, in essence, creating one of these things.

* A entity "view mode" is a mode by which a type of entity-thing can be viewed. For example, you already have a node. That same node can be seen in any of a number of modes, for example, Teaser, Full, Sidebar, Slide, etc, etc. 

The thing I find confusing about the mode name is the descriptor `view` which comes from the hook (or op) name, `hook_node_view`. It makes perfect sense for a developer, but not so much for a user who doesn't know the name of that hook. Especially with the Views module now in core, people are very likely to think these things have something to do with views, which is not (necessarily) the case.

Drupal 8 chose to use the term `Display mode` which I quite like, since it removes the confusion with Views module. However, as @klonos points out in this issue, it might still cause some confusion/association with views `Displays`.  

I've created another issue that handles only the text change just so people could take a look at exactly how much needs to be changed. It's a lot!!! After taking a stab at the text change I'm not entirely sure it's worth all that change, especially if people are still likely to get it confused with views-stuff. :/ https://github.com/backdrop/backdrop-issues/issues/2376

**Comment by klonos** [comment]
Fair enough about the amount of code change, but that should not be an excuse for blocking UX/UI improvements and clarifications of things that are likely to confuse people. We need to always keep in mind that Drupal is notorious of being "hard to learn". We should aim to change that in Backdrop CMS, *the easier version of Drupal* ( ™️ 😉 ).

**Comment by klonos** [comment]
...I quite liked what I said above. Made me imagine this dialog:

Webdev: What do you build your sites with? Drupal?
Me: Yep.
Webdev: Which version?
Me: The Backdrop version 😉

**Comment by jenlampton** [comment]
I totally agree, but if chaning "View mode" to "Display mode" is going to increase confusion with displays as it decreases confusion with Views, is it enough of a win?

**Comment by klonos** [comment]
No, no. I was not referring to using "Display mode" specifically. It may be that we'll end up using that, but we might as well come up with something entirely different. The specific wording is not the important thing here. The important thing is to figure a/the way to make things less confusing. We are all brilliant minds and brainstorming solutions collectively is where we excel. I'm very confident that we'll figure the best way ...or at least a better one.

**Comment by klonos** [comment]
We need to decide and fix this soon because the current status is very confusing:

![image](https://user-images.githubusercontent.com/2423362/27253494-16b501c4-53b9-11e7-8995-9653fa939193.png)

**Comment by stpaultim** [comment]
This comment in the forum seems relevant (@klonos pointed this out for me) to this issue.
https://forum.backdropcms.org/forum/adding-and-using-custom-display-modes

> In the Manage Displays page for a Content Type there is a link 'Add display mode'. Using this I can name and configure an alternative display mode but I cannot find how to use this instead of the default. Can the machine name be used as part of a URL?

> I have looked in our manual at 'Deep dive: Display Settings' but cannot find the answer there. Also it confused me regarding 'display mode' and 'view mode' - are these the same thing?

**Comment by stpaultim** [comment]
Is this issue still relevant given the changes made here: https://github.com/backdrop/backdrop-issues/issues/2376

This page (see screenshot) is much less confusing now. I'm tempted to close this issue. But, it's not clear if this issue is still advocating for a bigger change? @klonos 

![post___backdrop](https://user-images.githubusercontent.com/3144571/50944783-69212180-1457-11e9-95b6-e8558e9f025e.png)

**Comment by jenlampton** [comment]
This issue was for all the other places (like viwes UI) where it still says
"view mode"
